File: src/main/java/com/fifo/ticketing/domain/book/dto/BookSeatViewDto.java
package com.fifo.ticketing.domain.book.dto;

import com.fifo.ticketing.domain.seat.entity.SeatStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
@AllArgsConstructor
public class BookSeatViewDto {

    private Long seatId;
    private String seatNumber;
    private String grade;
    private int price;
    private SeatStatus seatStatus;
}

File: src/main/java/com/fifo/ticketing/domain/book/dto/BookSeatViewDto.kt
package com.fifo.ticketing.domain.book.dto

import com.fifo.ticketing.domain.seat.entity.SeatStatus

data class BookSeatViewDto(
    val seatId: Long,
    val seatNumber: String,
    val grade: String,
    val price: Int = 0,
    val seatStatus: SeatStatus?
) {
}